fragments/k8s/cdk/bundle-converged.yaml

```diff
@@ -0,0 +1,67 @@
+# This is an incomplete bundle fragment. Do not attempt to deploy.
+description: |-
+    A nine-machine Kubernetes cluster, appropriate for production. Includes a three-machine etcd cluster and three Kubernetes worker nodes.
+series: xenial
+services:
+  "kubernetes-master":
+    charm: "cs:~containers/kubernetes-master"
+    num_units: 1
+    options:
+      channel: 1.10/stable
+    constraints: "cores=2 mem=4G root-disk=16G"
+    annotations:
+      "gui-x": "800"
+      "gui-y": "850"
+  "kubeapi-load-balancer":
+    charm: "cs:~containers/kubeapi-load-balancer"
+    num_units: 1
+    expose: true
+    constraints: "root-disk=8G"
+    annotations:
+      "gui-x": "450"
+      "gui-y": "250"
+  "easyrsa":
+    charm: "cs:~containers/easyrsa"
+    num_units: 1
+    constraints: "root-disk=8G"
+    annotations:
+      "gui-x": "450"
+      "gui-y": "550"
+    to:
+     - kubernetes-master
```

Comment:
So the only thing converging in this bundle is easyrsa - is that the intent?

Reply:
for now, yes